packages/translate/test/index.js

```diff
@@ -462,44 +497,83 @@ describe('Translate', function() {
   });
 
   describe('request', function() {
-    it('should make the correct request', function(done) {
-      var userAgent = 'user-agent/0.0.0';
-
-      var getUserAgentFn = fakeUtil.getUserAgentFromPackageJson;
-      fakeUtil.getUserAgentFromPackageJson = function(packageJson) {
-        fakeUtil.getUserAgentFromPackageJson = getUserAgentFn;
-        assert.deepEqual(packageJson, require('../package.json'));
-        return userAgent;
-      };
+    describe('OAuth mode', function() {
+      var request;
 
-      var reqOpts = {
-        uri: '/test',
-        a: 'b',
-        c: 'd',
-        qs: {
+      beforeEach(function() {
```

Comment:
```sh
  1) Translate request API key mode should make the correct request:
     TypeError: Cannot read property 'call' of undefined
      at Translate.request (src/index.js:437:37)
      at Context.<anonymous> (test/index.js:576:19)
```

It would seem it is trying to use `Service#request` to make the request. My guess is that the `before` hooks are not happening in the expected order and the Translate instance using an API key gets overwrote by an instance using the standard credentials.